tardis/visualization/tools/sdec_plot.py

```diff
@@ -941,7 +1034,21 @@ def _plot_absorption_mpl(self):
         """Plot absorption part of the SDEC Plot using matplotlib."""
         lower_level = np.zeros(self.absorption_luminosities_df.shape[0])
 
-        elements_z = self.absorption_luminosities_df.columns.to_list()
+        # If the 'other' column exists then plot it as silver
+        if "other" in self.absorption_luminosities_df.keys():
+            upper_level = lower_level
+            lower_level = (
+                upper_level - self.absorption_luminosities_df.other.to_numpy()
+            )
+
+            self.ax.fill_between(
+                self.plot_wavelength,
+                upper_level,
+                lower_level,
+                color="silver",
+            )
```

Comment:
Is there any logical reason of putting this code before for loop that I'm not aware of? 

If not, could you please move it down after for loop i.e. L1070 because comments in L1053-1055 should remain in the start as they explain this upper_level - lower_level logic.

Reply:
I think the 'other' section should come first so it's plotted first. This keeps the silver part on the bottom, which I think makes more sense. Therefore I've moved the comments describing the upper - lower logic to here